src/Component.js

```diff
@@ -35,7 +35,7 @@ export class CopyToClipboard extends React.PureComponent {
     const result = copy(text, options);
 
     if (onCopy) {
-      onCopy(text, result);
+      onCopy(text, result, event);
```

Comment:
Why is this necessary? It is possible to use `onClick` callback and have the event